src/main/java/frc/robot/subsystems/BitBucketsSubsystem.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.BaseTalon;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Sendable;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Robot;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class BitBucketsSubsystem extends SubsystemBase
+{
+    protected final DriverStation driverStation;
+
+    protected final List<? extends BaseTalon> motorList;
+
+    public BitBucketsSubsystem()
+    {
+        this.setName(this.getClass().getSimpleName());
+
+        this.driverStation = DriverStation.getInstance();
+
+        this.motorList = new ArrayList<>();
+        this.addMotorsToList();
```

Comment:
I never liked the way this was implemented before. Do you think it would make more sense for each subsystem to return a list of all motors it manages rather than add to some motors list that is a protected var on the class?